<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>man_hinh_quan_ly_hoa_don</class>
 <widget class="QMainWindow" name="man_hinh_quan_ly_hoa_don">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="thong_ke_gr">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>811</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 212);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="check">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 255);</string>
     </property>
     <property name="text">
      <string>Hóa Đơn</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>700</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Đăng Xuất</string>
     </property>
    </widget>
    <widget class="QPushButton" name="main">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Trang chủ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="account">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>580</y>
       <width>171</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Tài Khoản</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>747</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Trở về</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quan_ly_hoa_don">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>251</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(0, 255, 255);</string>
     </property>
     <property name="text">
      <string>Quản Lý Hóa Đơn</string>
     </property>
    </widget>
    <widget class="QPushButton" name="user">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Người Dùng</string>
     </property>
    </widget>
    <widget class="QPushButton" name="thong_ke">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>241</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Thống Kê</string>
     </property>
    </widget>
    <widget class="QPushButton" name="menu">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>MENU</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hoa_don">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>30</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Hóa Đơn</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>370</y>
       <width>691</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="title">
      <string>Thông Tin Hóa Đơn</string>
     </property>
     <widget class="QLineEdit" name="ma_hoa_don_qlhd1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ten_sp_qlhd1">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ten_ban_qlhd1">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="so_luong_qlhd1">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>80</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="don_gia_qlhd1">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>80</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="tong_qlhd1">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>80</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="don_gia_qlhd2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>100</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ten_sp_qlhd2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ma_hoa_don_qlhd2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="so_luong_qlhd2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>100</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ten_ban_qlhd2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="tong_qlhd2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>100</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ma_hoa_don_qlhd4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="don_gia_qlhd3">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>120</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ten_sp_qlhd4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ten_sp_qlhd3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ma_hoa_don_qlhd3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ten_ban_qlhd4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="so_luong_qlhd3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>120</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="don_gia_qlhd4">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>140</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="so_luong_qlhd4">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>140</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="tong_qlhd4">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>140</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ten_ban_qlhd3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>120</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="tong_qlhd3">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>120</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Mã Hóa Đơn</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tên Sản Phẩm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tên Bàn</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>60</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Số Lượng</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>60</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Đơn giá</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>60</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tổng</string>
      </property>
     </widget>
     <zorder>ten_sp_qlhd1</zorder>
     <zorder>ten_ban_qlhd1</zorder>
     <zorder>so_luong_qlhd1</zorder>
     <zorder>don_gia_qlhd1</zorder>
     <zorder>tong_qlhd1</zorder>
     <zorder>don_gia_qlhd2</zorder>
     <zorder>ten_sp_qlhd2</zorder>
     <zorder>so_luong_qlhd2</zorder>
     <zorder>ten_ban_qlhd2</zorder>
     <zorder>tong_qlhd2</zorder>
     <zorder>ma_hoa_don_qlhd2</zorder>
     <zorder>ma_hoa_don_qlhd1</zorder>
     <zorder>don_gia_qlhd3</zorder>
     <zorder>ten_sp_qlhd4</zorder>
     <zorder>ten_sp_qlhd3</zorder>
     <zorder>ma_hoa_don_qlhd3</zorder>
     <zorder>ten_ban_qlhd4</zorder>
     <zorder>so_luong_qlhd3</zorder>
     <zorder>don_gia_qlhd4</zorder>
     <zorder>so_luong_qlhd4</zorder>
     <zorder>tong_qlhd4</zorder>
     <zorder>ten_ban_qlhd3</zorder>
     <zorder>tong_qlhd3</zorder>
     <zorder>ma_hoa_don_qlhd4</zorder>
     <zorder>label_17</zorder>
     <zorder>label_18</zorder>
     <zorder>label_19</zorder>
     <zorder>label_20</zorder>
     <zorder>label_21</zorder>
     <zorder>label_22</zorder>
    </widget>
    <widget class="QLineEdit" name="tim_kiem_hoa_don">
     <property name="geometry">
      <rect>
       <x>342</x>
       <y>141</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/picture/398288296_664763935842542_91204699368765392_n.png);
border-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>572</x>
       <y>137</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(189, 189, 189);</string>
     </property>
     <property name="text">
      <string>Tìm kiếm</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="add_image.qrc"/>
 </resources>
 <connections/>
</ui>
